platforms/ios/example/Shared/WysiwygAction+Utils.swift

```diff
@@ -55,6 +55,8 @@ extension WysiwygAction: CaseIterable, Identifiable {
             return .strikeThroughButton
         case .underline:
             return .underlineButton
+        case .inlineCode:
+            return .inlineCode
```

Comment:
I think this needs to be:

```suggestion
            return .inlineCodeButton
```

It wouldn't let me build the project any other way.

Reply:
My bad, didn't commit this as I was doing things with links in the same branch 🙈